docs/docker.md

```diff
@@ -0,0 +1,97 @@
+# Docker support
+
+The docker runtime is supported by kubespray and while the `dockershim` is deprecated to be removed in kubernetes 1.24+ there are alternative ways to use docker such as through the [cri-dockerd](https://github.com/Mirantis/cri-dockerd) project supported by Mirantis.
+
+Using `cri-dockerd` instead of `dockershim`:
+
+```yaml
+cri_dockerd_enabled: false
```

Comment:
How about adding
```
container_manager: docker
```
to understand what are necessary easily?

Reply:
I updated the docs in the latest push.